### DISCUSSION TITLE
`v2`-layout questions and comments - April 2024

### DISCUSSION THREAD

**Discussion by JC5** [firefly-iii/firefly-iii#8745] [General]
Hi there,

Please use this GitHub discussion to report all your Firefly III v2 layout issues and problems, and see if a fix is perhaps already available. Due to the volatile nature of software development, the v2 layout may be buggy or broken. As such the rule is: if it breaks you get to keep both parts. However, I am of course keen to fix these issues so please report them here.

This discussion will be closed and recreated every month, depending on the number of replies and issues etc.

### How do I enable the v2 layout?

To enable the v2 layout installation in your installation, please set `FIREFLY_III_LAYOUT=v2` in your environment variables. Be aware that pages may be broken, functionality is missing and data loss may occur when you use the layout to submit or change your data.

### Can I go back (temporarily) to the normal layout?

Each page has a 🏛️ -icon. Press it, and you will go back to the current layout.

### Can I go permanently back to the normal layout?

Yes. Set `FIREFLY_III_LAYOUT=v1`.

### How do I enable debug mode for easier debugging?

- [How to enable debug mode](https://docs.firefly-iii.org/how-to/general/debug/)

**Comment by JC5** [comment]
- @adyanth the missing `currency_id` should be fixed.
- @Tealk Thanks for the hint, I'm working to fix the boxes.

See also:

- [March 2024](https://github.com/orgs/firefly-iii/discussions/8615)
- [February 2024](https://github.com/orgs/firefly-iii/discussions/8491)
- [January 2024](https://github.com/orgs/firefly-iii/discussions/8360)

**Reply by RokSiEu** [reply]
I'm now on 6.1.13 and few issues from the start

1.  accounts are not loaded (assets, expense etc.)

![image](https://github.com/firefly-iii/firefly-iii/assets/2094036/6c3b284d-96f1-479b-bff4-b97ca77e9a00)

2. URL transactions/all is not working

![image](https://github.com/firefly-iii/firefly-iii/assets/2094036/fbefbec2-3b12-473b-bd03-82d5aee08422)

3. transaction list (expense) is quite messed up, also the table only shows the description of the transaction,everything else is blank

![image](https://github.com/firefly-iii/firefly-iii/assets/2094036/39ee074b-de48-4f4a-a8ba-bc4e8785b1f7)

**Comment by RokSiEu** [comment]


**Reply by JC5** [reply]
Which version of Firefly III are you on?

**Reply by RokSiEu** [reply]
6.1.7 and v2 of front end.

**Reply by JC5** [reply]
Ok start by upgrading to the latest release.

**Reply by RokSiEu** [reply]
Sorry, it seems my scheduler did not run and image wasn't updated for a while. 
In 6.1.13 I don't have this issues.

**Comment by arifin086** [comment]
Hi All, 

How do I edit the frontend of firefly-iii? Especially the login page as it does not have the registration or register link. I tried editing the login.blade.php but its not capturing anything. Also the register page, the I want to login does not work. Please help.

**Comment by xaiyadev** [comment]
Hello, is this still beeing worked on?
Any updates?

**Reply by JC5** [reply]
Yes, it is. These threads don't get much attention so I've stopped creating them. Work progresses steadily.

**Reply by xaiyadev** [reply]
That's awesome!
Can't wait for the finished/stable results
